win32ss/gdi/ntgdi/bitblt.c

```diff
@@ -1598,6 +1598,9 @@ NtGdiGetPixel(
 
         /* Delete the surface */
         GDIOBJ_vDeleteObject(&psurfDest->BaseObject);
+
+        /* The top byte is zero */
+        ulRGBColor &= 0x00FFFFFF;
```

Comment:
So it doesn't support returning an "alpha" channel in case the underlying bitmap format is "more" than 24-bits?

Reply:
I have to investigate why `gdiplus` can support alpha channel.

Reply:
`gdiplus` uses a special function named `GdipBitmapGetPixel`. That function does manually getting the pixel from the bitmap.

Reply:
> So it doesn't support returning an "alpha" channel in case

No, it doesn't.